examples/tutorials/hello-gltf/app.ts

```diff
@@ -37,7 +43,14 @@ export default class AppAnimationLoopTemplate extends AnimationLoopTemplate {
     modelSelector?.addEventListener('change', e => {
       const name = (e.target as HTMLSelectElement).value;
       this.loadGLTF(name);
-      window.localStorage.lastGltfModel = name;
+      window.localStorage['lastGltfModel'] = name;
+    });
+
+    Object.keys(this.options).forEach(id => {
```

Comment:
I think you should collect the DOM-aware code in a function below and accept a callback rather than messing wth the DOM here in the main function.

My motivation is to keep the main logic minimal, focused on luma API, and make it easy to cut out the HTML components if someone wants to clone this example